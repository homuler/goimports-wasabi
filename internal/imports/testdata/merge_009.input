// merge named import declarations
package main

import (
	"context"
	"fmt"
)
import f "fmt"

func main() {
	fmt.Println(context.Background())
	f.Printf("%v %v %v %v\n", x, y, w, z)
}
