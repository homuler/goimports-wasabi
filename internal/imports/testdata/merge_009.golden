// merge named import declarations
package main

import (
	"context"
	"fmt"
	f "fmt"
)

func main() {
	fmt.Println(context.Background())
	f.Printf("%v %v %v %v\n", x, y, w, z)
}
